src/typings/vue-i18n.ts

```diff
@@ -0,0 +1,11 @@
+import Vue from 'vue';
+
+declare module 'vue/types/vue' {
+  interface t {
+    (key: string, args?: any, raw?: boolean): string,
+  }
+
+  interface Vue {
+    t: t;
+  }
```

Comment:
We don't need an extra interface for a function:
```suggestion
  interface Vue {
    /**
     * Lookup a given string with the given arguments
     * @param raw if set, do not do HTML escaping.
     */
    t: (key: string, args?: Record<string, any>, raw?: boolean) => string,
  }
```
(I may have went a bit overkill there…)